packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -227,7 +255,7 @@ class SkiaGoldClient {
   /// result.
   ///
   /// The [testName] and [goldenFile] parameters reference the current
-  /// comparison being evaluated by the [FlutterSkiaGoldFileComparator].
+  /// comparison being evaluated by the [FlutterPostSubmitFileComparator].
   Future<bool> imgtestAdd(String testName, File goldenFile) async {
```

Comment:
@blasten, on master in post-submit, this currently returns true regardless of the result. For driver tests, did you want this to fail since we do not run pre-submit tests?